aten/src/ATen/native/Convolution.cpp

```diff
@@ -265,6 +265,20 @@ at::Tensor convolution(
                           ctx.benchmarkCuDNN(), ctx.deterministicCuDNN(), ctx.userEnabledCuDNN());
 }
 
+static inline std::vector<int64_t> convolution_expand_param_if_needed(
+  IntList &list_param, std::string param_name, size_t expected_dim) {
```

Comment:
Pass `IntList` by value.

Use `const char*` for param_name. Otherwise (at least on less recent GCC versions) it will create a new `std::string` every call.

Reply:
BTW, it's doubly bad to pass a non-const reference, because you can accidentally silently mutate the IntList in the enclosing scope, which is basically never what you want.